sys-fs/xfsprogs/files/xfsprogs-6.4.0-lfs64.patch

```diff
@@ -0,0 +1,645 @@
+From a283105bee3de7941244a75d35379cb22a3efbbb Mon Sep 17 00:00:00 2001
```

Comment:
Looks good, but upstream link please? (always!)

Reply:
Its from Alpine.

https://git.alpinelinux.org/aports/tree/main/xfsprogs/lfs64.patch?id=0c57de04936543b82b55b320951b3ed84f3d45fb

I shared it on #xfs @ OFTC some while ago where it was being considered if it was the correct fix or not, but I never heard what happened after that.

Reply:
This patch has her name on and includes justification for why glibc is fine with this change too. It's also the obviously-correct set of changes for the problem. That doesn't mean it's from Alpine.

Reply:
> Looks good, but upstream link please? (always!)

Where is the correct place to submit a patch for xfsprogs? I was looking at the XFS mailing list but I can't tell if that's the correct place for xfsprogs patches.

Reply:
yeah, it really is linux-xfs, I sent a patch for xfsprogs last month: https://lore.kernel.org/linux-xfs/20230626095048.1290476-1-sam@gentoo.org/T/#u

Reply:
Ping. Any success with upstreaming?